src/ng1/directives/splitter/splitter.directive.js

```diff
@@ -313,6 +322,20 @@ export default function splitter($compile, $timeout) {
                 elem[0].style.transition = "";
             }
 
+            function toggleDisabled(toggle) {
```

Comment:
Function names should be verb-phrases. `toggleDisabled` implies that disabled will be toggled to the opposite state when called. Also you do not need to pass `toggle` as a parameter since it is in scope.

````
setToggleDisabled(state: boolean)
````